src/view/store.ts

```diff
@@ -3,10 +3,7 @@ import { FluidModel } from '../model';
 import { Node } from '../model/types';
 import { useGetStore } from '../utils/hooks';
 
-const getDiceArray = (state: Record<string, Node>): Node[] =>
-  Object.keys(state).map((key: string) => ({ key, value: state[key].value }));
-
-const getLoadState = (model: FluidModel) => model.getAllNodes();
+type IDiceStore = Record<string, Node>;
```

Comment:
```suggestion
type IDiceState = Record<string, Node>;
```
since the store itself is not being defined by this interface but rather the state of the store